docs/release/qa.md

```diff
@@ -46,28 +46,42 @@ For both:
     ```sh
     docker image pull mathesar/mathesar-caddy:<version_number>
     docker image tag mathesar/mathesar-caddy:<version_number> mathesar/mathesar-caddy:latest
-    
+
     docker image pull mathesar/mathesar:<version_number>
     docker image tag mathesar/mathesar:<version_number> mathesar/mathesar:latest
     ```
 
+### Understanding Database connections
+
+#### Database types
+
+- **Mathesar Database** - The Django database used by Mathesar
+- **Data database** - The user database(s) being explored in Mathesar's UI
+
+#### Connection types
+
+- **Local** - On the host machine
+- **External** - Outside of the host machine
+- **Integrated** - Inside of Mathesar's docker image
+
 ### Tasks
 
 - [ ] Test installing the release from scratch by following our documentation
 - [ ] Test all different installation methods
-    - [ ] Docker compose
-        - [ ] All defaults, local
-        - [ ] Exposed on domain, DB managed by Mathesar
-        - [ ] Exposed on domain, DB preexisting for users, managed DB for Django
-        - [ ] Exposed on domain, DB preexisting for all data
-    - [ ] Docker integrated image
-        - [ ] All defaults, local
-        - [ ] Exposed on domain, DB managed by Mathesar
-        - [ ] Exposed on domain, DB preexisting for users, managed DB for Django
-        - [ ] Exposed on domain, DB preexisting for all data
-    - [ ] Build from scratch
-        - [ ] All defaults, local
-        - [ ] Exposed on domain, DB preexisting for all data
+    - [ ] [Docker compose](https://docs.mathesar.org/latest/administration/install-via-docker-compose/)
+        - [ ] Local using all default settings
+        - [ ] Exposed on domain with different database configurations:
+          - [ ] Integrated Mathesar, Integrated Data
+          - [ ] Integrated Mathesar, External Data
+          - [ ] External Mathesar, External Data
+          - [ ] External Mathesar, Integrated Data
```

Comment:
I don't think we should test this, if someone already has an external django db configured the integrated postgres server would be inaccessible as it would be turned off.